os/binfmt/binfmt_loadlibrary.c

```diff
@@ -119,6 +124,12 @@ int load_library(FAR const char *filename)
 	}
 #endif
 
+#ifdef CONFIG_ARMV7M_MPU
+	mpu_configure_library_region(g_mpu_region_nr++, bin->alloc[0], bin->textsize, true, true);
+	mpu_configure_library_region(g_mpu_region_nr++, bin->alloc[3], bin->rosize, true, false);
+	mpu_configure_library_region(g_mpu_region_nr++, bin->alloc[4], bin->ramsize, false, false);
```

Comment:
do you think using #defines for text ro & data indexes would be better ?

Reply:
Done